src/obstacle_detection.cpp

```diff
@@ -1,43 +0,0 @@
-#include "geometry_msgs/Point.h"
```

Comment:
Lets try not to pull things that don't build. I havent tested this version, but just as a general rule